src/web/src/components/SearchResults.tsx

```diff
@@ -56,7 +56,7 @@ type SearchResultProps = {
 const SearchResults = ({ text, filter }: SearchResultProps) => {
   const classes = useStyles();
   const prepareUrl = (index: number) =>
-    `${telescopeUrl}/query?text=${encodeURIComponent(text)}&filter=${filter}&page=${index}`;
+    `${postsServiceUrl}/query?text=${encodeURIComponent(text)}&filter=${filter}&page=${index}`;
```

Comment:
A lot of our current config URLs deal with adding the end-point (i.e., `/query`), see https://github.com/Seneca-CDOT/telescope/blob/master/src/web/src/config.ts#L15-L18.  I wonder if we should do the same, so you'd do:

`${postsServiceUrl}?text=${encodeURIComponent(text)}&filter=${filter}&page=${index}`;

Reply:
Another question: why are we doing a search (/query) at the posts URL?